MultiServer.py

```diff
@@ -1609,7 +1631,14 @@ async def process_client_cmd(ctx: Context, client: Client, args: dict):
         else:
             team, slot = ctx.connect_names[args['name']]
             game = ctx.games[slot]
-            ignore_game = ("TextOnly" in args["tags"] or "Tracker" in args["tags"]) and not args.get("game")
+
+            ignore_game = False
+            if not args.get("game"):
+                for tag in args["tags"]:
+                    if tag in _non_game_messages:
+                        ignore_game = True
+                        break
+
```

Comment:
I'm on the fence about the `for`/`else` that remyjette pointed out above, but this one I do think would be better using `any`

```python
ignore_game = not args.get("game") and any(tag in _non_game_messages for tag in args["tags"])
```

Reply:
yeah that'd be slightly more efficient (in my experience any works well for consolidating a lot of logical statements in an if statement)